Makefile

```diff
@@ -117,12 +117,16 @@ psql-crdb: kubectl-ctx-kind
 ## purge nats app and storage pvcs
 clean-nats:
 	kubectl delete statefulsets.apps nats && kubectl delete pvc nats-js-pvc-nats-0 nats-jwt-pvc-nats-0
-#
+
 ## set kube ctx to kind cluster
 kubectl-ctx-kind:
 	export KUBECONFIG=~/.kube/config_kind
 	kubectl config use-context kind-kind
 
+## update helm dependencies
```

Comment:
Can you add a comment on what this would do - so folks don't inadvertently the helm chart. Possibly even add a `readline "hit enter to proceed with updating the helm dependencies` or similar.

Reply:
helm dependency updates should break anything, they are needed and this target is mainly to be used by install and upgrade so you dont have to do it manually. I guess i can just add the one line in those targets instead.

Reply:
> helm dependency updates shouldnt break anything, they are needed 
I agree they are needed, but if a dev is working on some changes locally and runs `make update` would that bust the local helm chart changes?

Reply:
This was answered in another channel - local helm chart related changes are to be made in the project repo (Alloy/Flasher) and the steps in the docs included are to be followed to link the local helm chart. Then a `make install`, `make upgrade` should fetch the local chart.